### ISSUE TITLE
Uncertainty-Aware Self-Supervised Target-Mass Grasping of Granular Foods. (arXiv:2105.12946v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#7348]
Uncertainty-Aware Self-Supervised Target-Mass Grasping of Granular Foods. (arXiv:2105.12946v1 [cs.RO])<br>
https://ift.tt/34q88kY<br>
<p>Food packing industry workers typically pick a target amount of food by hand from a food tray and place them in containers. Since menus are diverse and change frequently, robots must adapt and learn to handle new foods in a short time-span. Learning to grasp a specific amount of granular food requires a large training dataset, which is challenging to collect reasonably quickly. In this study, we propose ways to reduce the necessary amount of training data by augmenting a deep neural network with models that estimate its uncertainty through self-supervised learning. To further reduce human effort, we devise a data collection system that automatically generates labels. We build on the idea that we can grasp sufficiently well if there is at least one low-uncertainty (high-confidence) grasp point among the various grasp point candidates. We evaluate the methods we propose in this work on a variety of granular foods -- coffee beans, rice, oatmeal and peanuts -- each of which has a different size, shape and material properties such as volumetric mass density or friction. For these foods, we show significantly improved grasp accuracy of user-specified target masses using smaller datasets by incorporating uncertainty.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
